src/main/java/com/reshiftsecurity/analytics/ReshiftAnalyticsAction.java

```diff
@@ -0,0 +1,23 @@
+package com.reshiftsecurity.analytics;
+
+import java.time.ZonedDateTime;
+
+public class ReshiftAnalyticsAction {
```

Comment:
I'm hesitant to name classes with 'Reshift' not sure it adds any value

Reply:
I only added those for less ambiguity while coding. Easier to get to the class I need.

Reply:
Any specific concern with is?

Reply:
First point is a general statement. I thinks its overused sometimes :) Last two points make sense. Will push  a change to remove "Reshift" from the names.